source/replication/site-replication-overview.rst

```diff
@@ -0,0 +1,462 @@
+            :align: center
+      
+      #. Select :guilabel:`Add Sites +`
+
+         .. image:: /images/minio-console/console-settings-site-replication-add.png
+            :width: 600px
+            :alt: MinIO Console's Add Sites for Replication screen
+            :align: center
+
+      #. Make the following entries:
+
+         :Access Key: `(required)` The user name to use for signing in to each site. Should be the same across all sites.
+
+         :Secret Key: `(required)` The password for the user name to use for signing in to each site. Should be the same across all sites.
+
+         :Site Name: An alias to use for the site name.
+
+         :Endpoint: `(required)` The URL or IP address and port to use to access the site.
+
+         To add additional sites beyond two, select the ``+`` button to the side of the last Site entry.
+
+      #. Click :guilabel:`Save`
+
+   .. tab-item:: Command line
+
+      #. Deploy three or more separate MinIO sites, using the same external IDP
+
+         Only one site can have any buckets or objects on it.
+         The other sites must be empty.
+
+      #. Configure an alias for each site
+
+         To check the existing aliases, use :mc-cmd:`mc alias list`.
+      
+         For example, for three MinIO sites, you might create aliases ``minio1``, ``minio2``, and ``minio3``.
+         
+         Use :mc-cmd:`mc alias set`
+      
+         .. code-block:: shell
+      
+            mc alias set minio1 https://minio1.example.com:9000 adminuser adminpassword
+            mc alias set minio2 https://minio2.example.com:9000 adminuser adminpassword
+            mc alias set minio3 https://minio3.example.com:9000 adminuser adminpassword
+      
+         or define environment variables
+      
+         .. code-block:: shell
+         
+            export MC_HOST_minio1=https://adminuser:adminpassword@minio1.example.com
+            export MC_HOST_minio2=https://adminuser:adminpassword@minio2.example.com
+            export MC_HOST_minio3=https://adminuser:adminpassword@minio3.example.com
+      
+      #. Add site replication configuration
+
+         List all existing replicated sites first, then list the new site(s) to add.
+         In this example, ``minio1``, ``minio2``, and ``minio3`` are already configured for replication.
+         The command adds minio4 and minio5 as new sites to add to the replication.
+         ``minio4`` and ``minio5`` must be empty.
+      
+         .. code-block:: shell
+         
+            mc admin replicate add minio1 minio2 minio3 minio4 minio5
+      
+      #. Query the site replication configuration to verify
+      
+         .. code-block:: shell
+         
+            mc admin replicate info minio1
+
+Modify a site's endpoint
+------------------------
+
+From time to time a replicated site's address may change.
+When this occurs, update the site's endpoint in the site replication's configuration.
```

Comment:
Personal nit - we don't need the soft lead in, we can get straight to the point.

"If a site in the replication config has changed or will change its hostname, you can modify the replication configuration to reflect the new hostname."